backend/src/controller/fhir_controller.py

```diff
@@ -58,7 +57,7 @@ def get_immunization_by_identifier(self, aws_event) -> dict:
             else:
                 raise UnauthorizedError()
         except UnauthorizedError as unauthorized:
-            return self.create_response(403, unauthorized.to_operation_outcome())
+            return create_response(403, unauthorized.to_operation_outcome())
```

Comment:
This is a good example of us trapping exceptions unnecessarily. We'd get the same result if we let the `UnauthorizedError` reach the top level handler, but with less logic in the controller

Reply:
The get by ID looks so much cleaner 🙌